addon/run-task.js

```diff
@@ -207,12 +221,18 @@ export function throttleTask(obj, name, timeout = 0) {
    */
 export function cancelTask(cancelId) {
```

Comment:
So this seems to imply we need to change the signature of this function to take an object in addition to the cancelId. Correct?